api/coreservice_test.go

```diff
@@ -230,6 +230,68 @@ func TestEstimateExecutionGasConsumption(t *testing.T) {
 
 }
 
+func TestTraceBlockByNumber(t *testing.T) {
+	require := require.New(t)
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+	svr, bc, _, ap, cleanCallback := setupTestCoreService()
```

Comment:
`setupTestCoreService(actionIndexer bool)` to control create action indexer or not
then here it should work with `setupTestCoreService(false)`